package.json

```diff
@@ -39,7 +39,7 @@
     "codecov": "nyc report --reporter=lcov && codecov",
     "clean": "git clean -fX ./dist ./*.xpi ./addon.env && npm run :clean:doc ",
     "distclean": "git clean -dfX",
-    "prepackage": "npm run-script clean && cross-env NODE_ENV=production npm run-script build",
+    "prepackage": "npm run-script clean && npm run-script build",
```

Comment:
If this gets moved out to the `travis.yml`, then I believe the Jenkins CI pipeline that does the extension build and signing (on production branch) won't get set to the right environment. 😔

Reply:
Where's the Jenkins script? Can't we update that?

Reply:
>Where's the Jenkins script? Can't we update that?
No, I don't think we can/should.

It's owned and managed by CloudOps (private mozilla-services repo) and [reused across other Test Pilot projects](https://github.com/mozilla/testpilot/blob/master/docs/development/hosting.md).

Since others rely on what's already there and assumptions are made around it, I don't think we should try to suggest any changes and instead keep making sure our "package" script lands what is expected.

Reply:
This deploy script seems... bad. But I guess it's just one of those things we have to live with.

Still, we should have a way to make a dev XPI so that it doesn't take a minute or more to get a  working XPI for testing. One option would be to make a `dev-package` script; another would be to ensure that the `production` branch always uses `NODE_ENV=production`.

Reply:
>Still, we should have a way to make a dev XPI so that it doesn't take a minute or more to get a working XPI for testing. 

Completely agreed. I figured that's what you were getting at here but wasn't sure.

However you think is best... I'm personally not sure how to approach the latter (tell `production` branch to use the right env) so former may be easiest for now? 😬